docs/content/latest/api/ysql/datatypes/type_array/literals/array-of-rows.md

```diff
@@ -0,0 +1,241 @@
+---
+title: The literal for an array of rows
+linkTitle: Array of rows
+headerTitle: The literal for an array of "row" type values
+description: The literal for an array of "row" type values
+menu:
+  latest:
+    identifier: array-of-rows
+    parent: array-literals
+    weight: 40
+isTocNested: true
+showAsideToc: true
+---
+
+We now combine the understanding of how to write the literal for an array of primitive values with that of how to write the literal for a _"row"_ type value.
+
+This section follows the same approach used in the section that explained the rules for the literal for an array of primitive values [here](../array-of-primitive-values/) and the section that explained the rules for the literal for a _"row"_ type value [here](../row/): first, it states the rules, and then it illustrates these with examples.
```

Comment:
```suggestion
This section follows the same approach used in the section that explained the rules for the literal for an array of primitive values [here](../array-of-primitive-values/) and the section that explained the rules for [the literal for a _"row"_ type value](../row/): first, it states the rules, and then it illustrates these with examples.
```

Reply:
Why did you change the second and not the first? I made a note to look at this aftre I have it back on my laptop.